### DISCUSSION TITLE
VM client

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1223] [Q&A]
Has anyone encountered these problems when downloading the latest VM clent ? 
1. when I connected to the crazyflie usually appear the python3 is not responding. 
![905736E3-415F-4D2B-8F82-770A657AECD5_1_102_o](https://github.com/bitcraze/discussions/assets/118436188/ea45a1af-27c4-4c62-9d07-0b9fcd775c10)
2. System often crash.

**Comment by ataffanel** [comment]
The VM tends to be very slow to communicate with the Crazyradio lately, something seems to have changed but we do not know what exactly.

The best way forward I think would be to install the client natively on your computer. This can be done on windows and Linux with ```pip install --upgrade cfclient```. On windows, the easiest way to install python is from the Windows store (type python in the command line and it should open automatically), but any other distribution of python will work fine.
